app/src/main/java/org/mozilla/tv/firefox/channels/content/SportsChannels.kt

```diff
@@ -32,13 +32,5 @@ fun ChannelContent.getSportsChannels(): List<ChannelTile> = listOf(
         setImage = setImage(R.drawable.tile_sports_cbs_sports),
         tileSource = TileSource.SPORTS,
         id = "cbsSports"
-    ),
```

Comment:
I think you need this bracket

Reply:
No, the final one needs to not have a comma.  This is also a .kt file, so if it were improperly matched it would fail to compile, so the test run would catch it.

Reply:
Oops sorry, I was referring to the closing bracket, but I didn't see it below.

Reply:
Easy mistake to make